version: '3'

services:
  kms:
    image: pykmsorg/py-kms:latest
    ports:
      - 1688:1688
    environment:
      - IP=0.0.0.0
      - SQLITE=true
      - HWID=RANDOM
      - LOGLEVEL=INFO
      - LOGSIZE=2
      - LOGFILE=/var/log/pykms_logserver.log
    restart: always
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - ./vlog:/var/log:rw
